README.md

```diff
@@ -25,7 +25,7 @@ In addition, minimally the following are required:
 
 On Debian/Ubuntu, this can be done using
 ```bash
-	apt-get -y install pkg-config libncursesw5-dev libnl-3-dev libnl-genl-3-dev
+apt -y install pkg-config libncursesw5-dev libnl-3-dev libnl-genl-3-dev
```

Comment:
Please keep `apt-get -y install`. This is the older variant and works on a _wider range_ of systems.

Reply:
Done.